ui-guidelines/css/resources-icons.css

```diff
@@ -0,0 +1,609 @@
+@font-face {
+    font-family: 'ez-icons';
+    src:
+        url('../fonts/ez-icons.ttf?nclra3') format('truetype'),
+        url('../fonts/ez-icons.woff?nclra3') format('woff'),
+        url('../fonts/ez-icons.svg?nclra3#ez-icons') format('svg');
+    font-weight: normal;
+    font-style: normal;
+}
+
+[class^="ez-icon-"], [class*=" ez-icon-"] {
+    font-family: 'ez-icons';
+    speak: none;
+    font-style: normal;
+    font-weight: normal;
+    font-variant: normal;
+    text-transform: none;
+    line-height: 1;
+
+    -webkit-font-smoothing: antialiased;
+    -moz-osx-font-smoothing: grayscale;
+}
+
+.icon-row {
+    display: flex;
+    justify-content: flex-start;
+    flex-wrap: wrap;
+    align-content: space-around;
+    margin-bottom: 2rem;
+}
+
+.icon-group {
+    margin: .5rem .5rem 0 0;
+}
+
+.icon-box {
+    height: 10rem;
+    width: 10rem;
+    background-color: #f3f3f3;
+    border: 1px solid transparent;
+    border-radius: 4px;
+    text-align: center;
+}
+
+.icon-size {
+    font-size: 32px;
+}
+
+.icon-label {
+    margin-top: -1rem;
+    padding: 0 .5rem;
+    font-size: 1.15rem;
+    font-family: 'Roboto', sans-serif;
+    font-weight: 400;
+    text-align: center;
+}
+
+/*icons*/
+.ez-icon-about:before {
+    content: "\e900";
+}
+
+.ez-icon-admin-dashboard:before {
+    content: "\e901";
+}
+
+.ez-icon-admin-panel:before {
+    content: "\e902";
+}
+
+.ez-icon-airtime:before {
+    content: "\e903";
+}
+
+.ez-icon-align-center:before {
+    content: "\e904";
+}
+
+.ez-icon-align-justify:before {
+    content: "\e905";
+}
+
+.ez-icon-align-left:before {
+    content: "\e906";
+}
+
+.ez-icon-align-right:before {
+    content: "\e907";
+}
+
+.ez-icon-alloy-h1:before {
```

Comment:
We should avoid to have `alloy` in the name. 
Why not using `oe` for everything related to online editor (`ez-oe-h1`)?